components/org.wso2.carbon.identity.api.idle.account.identification/org.wso2.carbon.identity.api.idle.account.identification.common/src/main/java/org/wso2/carbon/identity/api/idle/account/identification/common/util/Utils.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.idle.account.identification.common.util;
+
+import org.slf4j.MDC;
+
+import java.util.UUID;
+
+/**
+ * Util class.
+ */
+public class Utils {
+
+    /**
+     * Get correlation id of current thread.
+     *
+     * @return Correlation-id.
+     */
+    public static String getCorrelation() {
```

Comment:
Why do we need to separately handle the correlation ids for this API specifically? Shouldn't these be provided from a central place?

Reply:
Addressed with commit e52d2135a2cab52434b3a4ae45415981e1ab9011